pkg/gofr/migration/cassandra.go

```diff
@@ -0,0 +1,99 @@
+package migration
+
+import (
+	"time"
+
+	"gofr.dev/pkg/gofr/container"
+)
+
+type cassandraDS struct {
+	container.Cassandra
+}
+
+type cassandraMigrator struct {
+	container.Cassandra
+
+	migrator
+}
+
+func (cs cassandraDS) apply(m migrator) migrator {
+	return cassandraMigrator{
+		Cassandra: cs.Cassandra,
+		migrator:  m,
+	}
+}
+
+const (
+	CheckAndCreateCassandraMigrationTable = `CREATE TABLE IF NOT EXISTS gofr_migrations (
+version bigint,
```

Comment:
Please reformat this query into two lines instead of keeping each field into separate line.